strax/plugins/plugin.py

```diff
@@ -110,6 +110,8 @@ def __init__(self):
             raise ValueError(f"depends_on not provided for {self.__class__.__name__}")
 
         self.depends_on = strax.to_str_tuple(self.depends_on)
+        # Remove duplicates
+        self.depends_on = tuple(self.depends_on)
```

Comment:
`self.depends_on = tuple(set(self.depends_on))`

Reply:
But will this really kill the issue? https://github.com/AxFoundation/strax/issues/846